content/kubeone/main/guides/cis-benchmarking/cis1.8-kubeone1.7-k8s1.27/_index.en.md

```diff
@@ -0,0 +1,569 @@
++++
+title = "Benchmark on KubeOne-1.7.3 and Kubernetes-1.27"
+date = 2024-03-06T12:01:00+02:00
++++
+
+This benchmark guide helps you evaluate the security of a KubeOne cluster against each control in the CIS Kubernetes Benchmark.
+
+This guide corresponds to the following versions of KubeOne, CIS Benchmarks, and Kubernetes:
+
+| KubeOne Version  | Kubernetes Version | CIS Benchmark Version |
+| ---------------- | ------------------ | --------------------- |
+| 1.7.3               | 1.27                 | CIS-1.8                    |
+
+## Testing Methodology
+
+Each control in the CIS Kubernetes Benchmark was evaluated against a KubeOne cluster that was configured according to the accompanying hardening guide.
+
+These are the possible results for each control:
+
+**Pass:** The KubeOne cluster passes the audit outlined in the benchmark.
+
+**Not Applicable:** The control is not applicable to KubeOne because of how it is designed to operate. Details are provided for it.
+
+**Warn:** The control is manual in the CIS benchmark and it depends on the cluster's use-case or some other factor that must be determined by the cluster operator. These controls have been evaluated to ensure KubeOne doesn't prevent their implementation, but no further configuration or auditing of the cluster has been performed.
+
+**Fail:** The control will be fixed in a future KubeOne release.
+
+## Control Type: master
+### 1.1. Control Plane Node Configuration Files
+#### 1.1.1: Ensure that the API server pod specification file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.2: Ensure that the API server pod specification file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.3: Ensure that the controller manager pod specification file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.4: Ensure that the controller manager pod specification file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.5: Ensure that the scheduler pod specification file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.6: Ensure that the scheduler pod specification file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.7: Ensure that the etcd pod specification file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.8: Ensure that the etcd pod specification file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.9: Ensure that the Container Network Interface file permissions are set to 600 or more restrictive (Manual)
+
+**Result:** 🟡 Warn
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
```

Comment:
What's the difference between Warn and Fail? Should we maybe explain that the beginning of the document?

Reply:
Actually, this is explained on top of the document, but I can update the wording.

Reply:
No, I just missed it for some reason. Sorry 🤦🏻